### PR TITLE
Ctr 320

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

File: app/views/components/add_list_row.scala.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@(
+    name: String,
+    index: Int,
+    mode: Mode
+)(implicit messages: Messages)
+
+
+<li class="tabular-data__entry tabular-data__entry--grouped divider--bottom">
+    <div id="add-list-@index-answer" class="tabular-data__data tabular-data__data--3-12">@name</div>
+
```
Comment (vinnicombe) [inline app/views/components/add_list_row.scala.html]: 3 + 2 + 2 = 7

File: test/views/AnyOtherCompanyBenefitsViewSpec.scala [hunk 2]
```diff
@@ -67,11 +67,11 @@ class AnyOtherCompanyBenefitsViewSpec extends YesNoViewBehaviours {
 		}
 
 		"list item must have change buttons" in {
-			doc.getElementById("component-answer-list").text.contains("Change qwerty") mustBe true
+			doc.getElementById("component-answer-list").text.contains("Change") mustBe true
 		}
```
Comment (vinnicombe) [inline test/views/AnyOtherCompanyBenefitsViewSpec.scala]: Is the user answer still in the hidden text for screen readers?

File: app/controllers/AnyOtherTaxableIncomeController.scala [hunk 3]
```diff
@@ -66,11 +71,14 @@ class AnyOtherTaxableIncomeController @Inject()(appConfig: FrontendAppConfig,
     implicit request =>
       val result: Option[Future[Result]] = for {
         taxYear: SelectTaxYear <- request.userAnswers.selectTaxYear
```
Comment (vinnicombe) [inline app/controllers/AnyOtherTaxableIncomeController.scala]: Make these for yield patterns consistent

File: test/views/AnyOtherCompanyBenefitsViewSpec.scala (hunk 2 above)
Comment (MatthewAlexanderBeck) [reply test/views/AnyOtherCompanyBenefitsViewSpec.scala]: Yup

**Pull request merged**